contrib/babelfishpg_tsql/src/hooks.c

```diff
@@ -342,11 +347,22 @@ UninstallExtendedHooks(void)
 	modify_RangeTblFunction_tupdesc_hook = prev_modify_RangeTblFunction_tupdesc_hook;
 	fill_missing_values_in_copyfrom_hook = prev_fill_missing_values_in_copyfrom_hook;
 	check_rowcount_hook = prev_check_rowcount_hook;
+	transactionStmt_hook = prev_transactionStmt_hook;
 }
 
 /*****************************************
  * 			Hook Functions
  *****************************************/
+static void
```

Comment:
we should have boolean return type to control engine side execution based on whether we call PLTsqlProcessTransaction or not